src/core/env/dev/terraform.tfvars

```diff
@@ -135,6 +135,66 @@ eventhubs = [
         manage = false
       }
     ]
+  },
+  {
+    name              = "pdnd-io-cosmosdb-message"
+    partitions        = 3
```

Comment:
Maybe we could increase the partition to 32 to easier the reading from PDND (message cardinality is really high)

Reply:
These are DEV topics. I thought it was not necessary having too much partitions.

Reply:
I also reduced retention down to 1 day

Reply:
Retention is ok.
Partition could be reduced only if we do not need to perform any performance/stress test

Reply:
You're right, I didn't consider this opportunity. Switched back to 30 (multiple of 3, as PDND guys asked us)